pages/community/code-of-conduct.md

```diff
@@ -0,0 +1,122 @@
+
+If a person is found to violate the US-RSE Association Code of Conduct, the Steering Committee may take any action they deem appropriate
+to resolve the situation, including warning the person in question, removing them from a specific activity, banning them from further
+participation in activities (in person or online), or expulsion from the US-RSE Association.
+
+<br>
+
+## Standards for Behavior
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Please always be
+kind and courteous. There’s never a need to be mean, rude, or disrespectful. Thank you for helping make this a welcoming, friendly
+community for all.
+
+We, the members of the US-RSE Association, strive to:
+
+* Be empathetic, respectful, welcoming, friendly, and patient. We remember that every US-RSE event, project, and program is crafted by
+human beings who deserve to be treated with kindness and empathy. We work together to resolve conflict and assume good intentions. We
+may all experience some frustration from time to time, but we do not allow frustration to turn into a personal attack. A community where
+people feel uncomfortable or threatened is not a productive one.  
+* Be collaborative. Our work depends on the participation of many people, and in turn others depend on our work. Effective and friendly collaboration enables everyone to better achieve their goals.  
+* Be careful in the words that we choose. We are careful and respectful in our communication, and we take responsibility for our own speech. Be kind to others. Do not insult or put down other members of the community.
+
+<br>
+
+## Unacceptable Behavior
+We, the members of the US-RSE Association, are committed to making participation in this community a harassment-free experience.
+
+We will not accept harassment or other exclusionary behaviors, such as:
+* The use of sexualized language or imagery  
+* Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)  
+* Posting sexually explicit or violent material
+* Violent or intimidating threats or language directed against another person or group
+* Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+* Sexist, racist, or otherwise discriminatory jokes and language
+* Trolling or insulting and derogatory comments
+* Written or verbal comments which have the effect of excluding people on the basis of membership in a specific group, including level
+of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race,
+ethnicity, age, religion, or nationality
+* Public or private harassment
+* Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent
```

Comment:
If someone feels harassed, what option do they have other than report it privately? You definitely are correct that the third bullet is a bit contradictory. Either we remove it entirely (maybe not the best solution) or add a clause to say that it doesn't apply if the individual is reporting a case to the committee.  Your suggestion about intent to harm is also good, but then it could be fairly subjective about what constitutes that.

Reply:
I agree that "intent to harm" is subjective and probably too strong. Open to other suggestions. We could probably drop the "intent to harm" phrase but leave in the clause about reporting to the committee.

Reply:
I'm not sure the right way to resolve this either.  I'd sort of resigned to leave it in as others seemed to think it was worthwhile.  I feel that as a code of conduct we could take this out, rather than put in clauses.  If someone is sharing private communication with the intent to harm then that strikes me as violating other parts of the CoC directly.

Reply:
I'm trying to think of a case where someone violates the spirit of this and yet isn't engaging in "public or private harassment" and isn't acting "unethical or unprofessional" - which are both also explicitly disallowed.

Reply:
Isn't it sort of implied that if you are reporting a violation, you might be reporting private communication? I didn't really think otherwise until I was reading the comments here.

Reply:
I'm on the same line of thinking.  What's the downside of just taking it out entirely? Can someone think of a specific scenario, that we want to prevent, when this bullet would be the sole basis of violating the CoC?

Reply:
Removed the bullet. We can always add it back in at a later date.